### PR TITLE
Desktop feature

### PR DESCRIPTION
**Pull request by Gunjuzone**
### Tablet and Desktop Views

In this pull request, my mobile portfolio is styled to display in tablet and desktop view using media query of min-width of 600, 767 and 919.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,75 +8,89 @@
...
 			<p>I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
 				don’t hestiate to contact me.</p>
+      </div>
 				<ul class="links">
 					<li class="lists"><a href="#"> <img src="Assets/github.png" alt=""></a></li>
 					<li class="lists"><a href="#"> <img src="Assets/Linkedin.png" alt=""></a></li>
 					<li class="lists"><a href="#"> <img src="Assets/Vector.png" alt=""></a></li>
 					<li class="lists"><a href="#"> <img src="Assets/twitter.png" alt=""></a></li>
 					<li class="lists"><a href="#"> <img src="Assets/medium.png" alt=""></a></li>
 				</ul>
```
Comment (Shaher-11) [inline index.html:37]: Kindly increase the size of the image on the left of the section as shown in the design below 

#### Original design
right1

#### Your Project
wrong1

File: index.html [hunk 2]
```diff
@@ -8,75 +8,89 @@
...
-        <button class="btn-skills-project">See Project</button>
-      </div>				
-      
-			</div>
-		</div>
+  </section>
 
-		<div class="data">
+  <section class="recent-profession">
+          <h2 class="recent-header">My Recent Works</h2>
```
Comment (Shaher-11) [inline index.html:41]: kindly align the text here to the left 

#### Original design
wrong2

##### Your Project
right2

File: index.html [hunk 3]
```diff
@@ -8,75 +8,89 @@
...
+        </ul>
+        <div class="btn-skls">
+          <button class="btn-skills-project">See Project</button>
+        </div>        
+        </div>
+      </div>
+      
+    </section>
+    
+  <section class="skills-grid">
```
Comment (Shaher-11) [inline index.html:67]: Reduce the width of the section and replace the see project button with the languages
#### Original design 
right3

#### Your Project
wrong3

File: index.html [hunk 4]
```diff
@@ -142,97 +157,94 @@ <h2 class="datatext">Profesional Art Printing Data</h2>
...
+        <h2>About me</h2>
+          
+        <p class="soft-dev">Hello I’m a software developer! I can help you build a product,feature 
+          or website look through some of my work and experience!If you like what you see 
+          and have a project you need coded, don’t hestiate to contact me.
+        </p>
+    </div>
       
     <div class="btn-mid-about">
       <button class="btn-see-about">Get My Resume</button>
```
Comment (Shaher-11) [inline index.html:182]: Add the hover effect to the button

File: index.html [hunk 5]
```diff
@@ -142,97 +157,94 @@ <h2 class="datatext">Profesional Art Printing Data</h2>
...
+
+      <div class="about-me about-desktop">
+      <div>
+        <h2>About me</h2>
+          
+        <p class="soft-dev">Hello I’m a software developer! I can help you build a product,feature 
+          or website look through some of my work and experience!If you like what you see 
+          and have a project you need coded, don’t hestiate to contact me.
+        </p>
+    </div>
```
Comment (Shaher-11) [inline index.html]: Reduce the space between the text and image on the left

File: index.html [hunk 6]
```diff
@@ -142,97 +157,94 @@ <h2 class="datatext">Profesional Art Printing Data</h2>
...
+        <textarea id="msg" name="user_message" placeholder="Enter text here" maxlength="500" required></textarea>
+      </li>
+
+      <li>
+          <button class="button-submit" type="submit">Get in touch</button>
+      </li>
+
+    </ul>
 
     </form>
```
Comment (Shaher-11) [inline index.html:249]: Reduce the width of the form as shown below 

#### Original design
right4

#### Your project
wrong4

**Review by Shaher-11 — CHANGES_REQUESTED** [review]
#### Hello @Gunjuzone,

### Great job so far!
You have been doing an incredible  job on the project so far:clap::clap:
The project is very well structured:smiley::smiley:
However, we still need to make a few changes to get the project approved:muscle:

 ## Required Changes ♻️

- [ ] Check the comments below:point_down:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Happy Coding!👏👏👏

File: index.html (hunk 5 above)
Comment (Gunjuzone) [reply index.html]: Hello, the picture of this area was not included. I dont understand as I only have an imafe on the right, not left. Could you offer more explanation as to what I am to do ?

File: index.html (hunk 1 above)
Comment (Davidosky007) [reply index.html:37]: fixed 👍

File: index.html (hunk 4 above)
Comment (Davidosky007) [reply index.html:182]: fixed 👍

File: index.html (hunk 2 above)
Comment (Davidosky007) [reply index.html:41]: fixed 👍

File: index.html (hunk 3 above)
Comment (Davidosky007) [reply index.html:67]: I recommend you fix the difference as the previous reviewer pointed out.
* Add the proper project image and styling (the text on top of the image should be visible) for each project

File: index.html (hunk 5 above)
Comment (Davidosky007) [reply index.html]: This is what the previous reviewer was trying to point out
* The about me section should be properly aligned with the languages section
Yours :
Screenshot (86)

Figma design: 

Screenshot (87)

File: index.html (hunk 6 above)
Comment (Davidosky007) [reply index.html:249]: fixed 👍

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi, @Gunjuzone, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
cr2 gif_almost-there

 ## Required Changes ♻️
* Nice job 👍 with all the requested changes you have made so far, but they are still some changes to be made, please check the comments below.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 3 above)
Comment (Gunjuzone) [reply index.html:67]: ## @Davidosky007,

Thanks for your review, I have been able to add proper project image and styling, below is a screenshot of how it is rendering on my desktop with the width of 30% using flex property. I would like clarification if there is anything I need to add more, I would like to implement all changes before I submit them for review as I only have one more review. 
Thanks.
cardscreen
 
Githubimg

**Review by Shaher-11 — APPROVED** [review]
#### Hello @Gunjuzone,

# Congratulations! 🎉 
#### Your project is approved!  There is nothing else to say other than... it's time to merge it :muscle: :muscle::muscle:

### Optional Suggestions💡
_The following point is optional to implement however it is highly recommended!_:smile::smile:

- Kindly make sure the space is equal on both sides in the projects section 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

**Pull request merged**